lvgl/src/mem.rs

```diff
@@ -69,14 +69,33 @@ impl<T> AsMut<T> for Box<T> {
     }
 }
 
+fn mem_info() -> lvgl_sys::lv_mem_monitor_t {
```

Comment:
Probably just need to make safe bindings for this simple function and add to the public API. It's useful for app debugging.